include/mfmg/adapters_dealii.hpp

```diff
@@ -156,7 +156,7 @@ class Adapter<DealIIMeshEvaluator<dim, VectorType>>
                    std::shared_ptr<boost::property_tree::ptree> params)
   {
     AMGe_host<mesh_type::dimension(), mesh_evaluator_type, vector_type> amge(
-        comm, mesh._dof_handler);
+        comm, mesh._dof_handler, params->get("eigensolver: arpack", true));
```

Comment:
Would prefer a string parameter "eigensolver: type" or similar, with possible values "arpack" or "lapack". So that in the future, we won't have to change it when we introduce our own solver.